autoarray/plot/mat_wrap/mat_plot.py

```diff
@@ -899,3 +912,57 @@ def _plot_voronoi_mapper(
         if not self.is_for_subplot:
             self.output.to_figure(structure=None, auto_filename=auto_labels.filename)
             self.figure.close()
+
+    def _plot_delaunay_mapper(
+        self,
+        mapper: MapperDelaunay,
+        visuals_2d: Visuals2D,
+        auto_labels: AutoLabels,
+        source_pixelilzation_values=None,
+    ):
+
+        extent = self.axis.config_dict.get("extent")

```

Comment:
There is an issue with the visualization of Delaunay source plane aspect ratios, which I will fix.